include/CLine3D.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include "TVector3.h"
+
+class CLine3D
+{
+ private:
+  TVector3 _point;
```

Comment:
I'm skeptical about relying on external software, including ROOT. Is it truly necessary to use a type that inherits from TObject as a member of a class that is not inherited from TObject? or are we only utilizing a few features that we could implement ourselves?

In addition, TVector3 is not recommended by ROOT itself...(read "Attention" in https://root.cern.ch/doc/master/classTVector3.html)